visivo/parsers/mkdocs_utils/nav_configuration_generator.py

```diff
@@ -0,0 +1,235 @@
+def _get_ref(field_data):
```

Comment:
Why are these "private"?

Reply:
I made the functions that we're just used within the nav_configuration_generator not being imported into the mkdocs.py module private. Does that make sense to do or should they just be public too?